h2o-r/h2o-package/R/kmeans.R

```diff
@@ -36,6 +36,8 @@
 #' @param categorical_encoding Encoding scheme for categorical features Must be one of: "AUTO", "Enum", "OneHotInternal", "OneHotExplicit",
 #'        "Binary", "Eigen", "LabelEncoder", "SortByResponse", "EnumLimited". Defaults to AUTO.
 #' @param export_checkpoints_dir Automatically export generated models to this directory.
+#' @param cluster_size_constraints Specify how many points should be at least in each cluster. The length of constraints array has to be same as
```

Comment:
minor edit:
Specify how many points should be at least in each cluster. The length of constraints array must be the same as the number of clusters (experimental).